chain-signatures/node/src/rpc.rs

```diff
@@ -432,6 +447,9 @@ async fn try_publish_near(
     crate::metrics::SIGN_LATENCY
```

Comment:
Since we have many sign metrics, can we rename this SIGN_LATENCY_TOTAL or something similar?